src/Castle.Core.Tests.WeakNamed/Castle.Core.Tests.WeakNamed.csproj

```diff
@@ -36,6 +36,7 @@
 		<PackageReference Include="System.Collections.Specialized" Version="4.3.0" />
 		<PackageReference Include="System.ComponentModel" Version="4.3.0" />
 		<PackageReference Include="System.ComponentModel.TypeConverter" Version="4.3.0" />
+		<PackageReference Include="System.Runtime.CompilerServices.VisualC" Version="4.3.0" />
```

Comment:
Needed to bring in the `IsLong` type (used by some modreq/modopt-related tests) in the `netstandard1.x` targets.